### ISSUE TITLE
I hope to add a host.

### ISSUE THREAD

**Issue by Stanley-baby** [meistrari/prompts-royale#7]
I hope to be able to add custom proxies for API options.

**Comment by henrycunh** [comment]
As in a different base URL for the OpenAPI or a proxy in the HTTP client?

**Comment by Stanley-baby** [comment]
Yes, it's similar to the AI plugin in Google Chrome. In the custom API interface, there are two options: {api key} and {host}. I don't know if you can understand it with this description.😅

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: henrycunh ***@***.***>
发送时间: Monday, July 17, 2023 8:16:02 PM
收件人: meistrari/prompts-royale ***@***.***>
抄送: Stanley-baby ***@***.***>; Author ***@***.***>
主题: Re: [meistrari/prompts-royale] I hope to add a host. (Issue #7)


As in a different base URL for the OpenAPI or a proxy in the HTTP client?
